homeassistant/components/lutron_caseta/light.py

```diff
@@ -83,3 +83,10 @@ async def async_turn_off(self, **kwargs):
     def is_on(self):
         """Return true if device is on."""
         return self._device["current_state"] > 0
+
+    @property
+    def unique_id(self):
+        """Return a unique identifier if serial number is None."""
+        if self.serial is None:
+            return f"light_{self._bridge_unique_id}_{self.device_id}"
+        return super().unique_id
```

Comment:
I think this is fine, but it would be good to get a test for this.

Reply:
Ok, no problem.  I'll figure out how to add a test for it.

Reply:
Wow, not sure where to start on creating this test.  
I've read through this: https://developers.home-assistant.io/docs/development_testing/ but it's a bit over my head.

Can you point me towards other documentation to help create a test for this?